_includes/installation-sb.html

```diff
@@ -9,7 +9,15 @@
     <h2>Table of Contents</h2>
   </header>
   <ul class="link-list nav">
-    <li><a href="#hardware">Get Your Hardware <small>(Connected)</small></a></li>
+    <li><a href="#started">Getting Started with Signal K</a></li>
```

Comment:
A &amp;nbsp; is generally a good idea between Signal and K, especially at the end of the line so we don't get dangling Ks.